src/server/http.rs

```diff
@@ -0,0 +1,164 @@
+use std::{sync::Arc, net::SocketAddr};
+use axum::{Router, routing::{get, delete, put}, http::StatusCode, response::IntoResponse, Json, extract::{State, Path}};
+
+use super::actor::runtime::{ActorRuntime, ActorTypeRegistration};
+use super::super::client::TonicClient;
+
+
+pub struct DaprHttpServer {
+    actor_runtime: Arc<ActorRuntime>,
+}
```

Comment:
_This comment is more for my own understanding to reconcile implementing existing  building blocks in this SDK, and implementing the actors building block._

Is this intended to be a new pattern for launching the dapr server?

Eg: there's the [existing `tonic` way](https://github.com/dapr/rust-sdk/blob/00ca92b817766b5171daabe9add8e2f267febb25/examples/pubsub/subscriber.rs#L80-L83) for the AppCallback. Is this the Hyper (via Axum) counterpart?

On that note, is there something special about actors that makes it different to launch than the existing AppCallback way? I haven't had a good look at the latest protos to see what is defined there (yet). _I also asked a variation of this further up, so maybe the answer will be up there_.

Reply:
Actors only support callbacks via HTTP today

Reply:
Yes, dapr only supports Http for actors... but this could also be a starting point for all other Http callbacks.